mibidata/mibi_image.py

```diff
@@ -617,25 +542,20 @@ def append(self, image):
             image: A MibiImage.
 
         Raises:
-            ValueError: Raised if
-
-                * The image has any channels already present on
-                  instance to which it is being appended.
-                * The channels to be appended do not match the form of the
-                  channels of the original image.
+            ValueError: Raised if the image has any channels already present on
+                instance to which it is being appended.
         """
-        og_channels = list(self.channels)
-        new_channels = list(image.channels)
-        if set(og_channels).intersection(set(new_channels)):
+        if set(self.channels).intersection(set(image.channels)):
             raise ValueError('Images contain overlapping channels.')
-        if MibiImage.channels_is_list_of_tuples(og_channels):
-            if not MibiImage.channels_is_list_of_tuples(new_channels):
-                raise ValueError('Channels to be appended must match form of '
+        if all((isinstance(c, tuple) and len(c) == 2 for c in self.channels)):
+            if not all((isinstance(c, tuple) and len(c) == 2
+                        for c in image.channels)):
+                raise ValueError('Channels to be appended must match form as '
```

Comment:
You removed from the docstring that this ValueError could be raised for

Reply:
Good catch, I was over-eager in my revert.